preliz/tests/test_posterior_to_prior.py

```diff
@@ -12,6 +14,10 @@
 
 def test_p2p():
     pz.posterior_to_prior(model, idata)
-    pz.posterior_to_prior(model, idata, alternative="auto")
+    pattern = r'\s*b = pm.\x1b\[1mGamma\x1b\[0m\("b",\s*alpha\s*=\s*([+-]?\d*\.?\d+),\s*beta\s*=\s*([+-]?\d*\.?\d+)\)'
+    assert bool(re.search(pattern, pz.posterior_to_prior(model, idata, alternative="auto")))
     pz.posterior_to_prior(model, idata, alternative=[pz.LogNormal()])
-    pz.posterior_to_prior(model, idata, alternative={"b": [pz.Gamma(mu=0)]})
+    pattern = r'\s*b = pm.\x1b\[1mGamma\x1b\[0m\("b",\s*mu\s*=\s*([+-]?\d*\.?\d+),\s*sigma\s*=\s*([+-]?\d*\.?\d+)\)'
```

Comment:
Not against your proposal, but what if test something simpler like`'Gamma\x1b[0m("b", alpha=' in ...` vs `'Gamma\x1b[0m("b", mu=' in ...`

Reply:
It sounds good, I have made the change